corehq/apps/reports/forms.py

```diff
@@ -243,28 +244,54 @@ class EmailReportForm(forms.Form):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
+        if get_bootstrap_version() == BOOTSTRAP_5:
```

Comment:
I'd be more inclined to do inline B3/B5 checks instead of making separate functions. I'm a little worried that people will update one but not the other. It looks like there are only 2-3 different lines in a 20-line function (the `field_class` in B3, the `data_dismiss`/`data_bs_dismiss`, and the `FormActions`).